tomls/omnibus-snaxchain-testnet/governance/councils-mothership.toml

```diff
@@ -1,7 +1,7 @@
 [clone.governance_ambassador_council]
 source = "<%= settings.governance_package %>"
-targetPreset = "ambassador"
-options.salt = "<%= settings.salt %>"
+target = "<%= settings.governance_package %>@ambassador"
```

Comment:
Do we not use targetPreset anymore?

Reply:
Yes. it was needed because cannon did not work with mix of template + string before, but targetPreset is technically deprecated (but works!)